pbj-core/pbj-runtime/src/main/java/com/hedera/pbj/runtime/io/buffer/Bytes.java

```diff
@@ -154,6 +158,34 @@ void writeTo(@NonNull final ByteBuffer dstBuffer) {
         dstBuffer.position(dstBuffer.position() + length);
     }
 
+    /**
+     * Package private helper method for efficient copy of
+     * our data into an OutputStream without creating a defensive copy
+     * of the data. The implementation relies on a well behaved
+     * OutputStream that doesn't modify the buffer data.
+     *
+     * @param outStream the OutputStream to copy into
+     */
+    public void writeTo(@NonNull final OutputStream outStream) {
+        try {
+            outStream.write(buffer);
```

Comment:
This is broken as it does not pay attention to start & length fields. Bytes is sometimes a subset of a shared byte[]

Reply:
Fixed.